wolfcrypt/src/port/atmel/atmel.c

```diff
@@ -603,7 +603,7 @@ int atcatls_create_pms_cb(WOLFSSL* ssl, ecc_key* otherKey,
         }
 
         ret = atmel_ecc_create_pms(tmpKey.slot, peerKey, out);
-        *outlen = ATECC_SIG_SIZE;
+        *outlen = ATECC_KEY_SIZE;
```

Comment:
Thank you for spotting this bug. You are correct. This should be returning 32 bytes here.